### PR TITLE
SC: change a option SC_TX_PERIOD to SC_ANCHORING_PERIOD

### PR DESCRIPTION
**Pull request by jeonghonah**
## Proposed changes

- Minor fix for a option name:
    - SC_TX_PERIOD --> SC_ANCHORING_PERIOD

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: build/packaging/linux/bin/kscnd [hunk 1]
```diff
@@ -177,7 +177,7 @@ start() {
     fi
 
     if [[ ! -z $SC_SUB_BRIDGE ]] && [[ $SC_SUB_BRIDGE -eq 1 ]]; then
-        OPTIONS="$OPTIONS --subbridge --subbridgeport $SC_SUB_BRIDGE_PORT --chaintxperiod $SC_TX_PERIOD --chaintxlimit $SC_TX_LIMIT "
+        OPTIONS="$OPTIONS --subbridge --subbridgeport $SC_SUB_BRIDGE_PORT --chaintxperiod $SC_ANCHORING_PERIOD --chaintxlimit $SC_TX_LIMIT "
```
Comment (KimKyungup) [inline build/packaging/linux/bin/kscnd]: Could you change `chaintxperiod` to `anchroingperiod`, too? =)

Comment (jeonghonah) [reply build/packaging/linux/bin/kscnd]: Good. How about apply the change in the next PR? The change will affect binary and node-tests not only packaging related files.

Comment (KimKyungup) [reply build/packaging/linux/bin/kscnd]: OK. :)

File: build/packaging/linux/conf/kscnd.conf [hunk 2]
```diff
@@ -36,12 +36,12 @@ WS_ORIGINS="*"
 # service chain options setting
 SC_MAIN_BRIDGE=0 # if this is set, the following options will be used.
 SC_MAIN_BRIDGE_PORT=50505
-SC_MAIN_BRIDGE_INDEXING=0 # this option will be deprecated.
+SC_MAIN_BRIDGE_INDEXING=0
 
-SC_SUB_BRIDGE=1
-SC_SUB_BRIDGE_PORT=50506    # if main-bridge and sub-bridge on same instance, us different port with main-bridge.(main: 50505, sub:50506)
+SC_SUB_BRIDGE=0
+SC_SUB_BRIDGE_PORT=50506  # if main-bridge and sub-bridge on same instance, us different port with main-bridge.(main: 50505, sub:50506)
```
Comment (laggu) [inline build/packaging/linux/conf/kscnd.conf]:
```suggestion
SC_SUB_BRIDGE_PORT=50506  # if main-bridge and sub-bridge on same instance, use different port with main-bridge.(main: 50505, sub:50506)
```

Comment (jeonghonah) [reply build/packaging/linux/conf/kscnd.conf]: Fixed.

**Pull request merged**